tests/conftest.py

```diff
@@ -84,6 +84,18 @@ def metadata_db(app):
         created_at TIMESTAMP NOT NULL,
         updated_at TIMESTAMP NOT NULL
     )""")
+    rds.execute(f"""CREATE TABLE IF NOT EXISTS {rds_schema}.canvas_api_import_job_status
+    (
+        job_id VARCHAR NOT NULL,
+        course_id VARCHAR NOT NULL,
+        term_id VARCHAR NOT NULL,
+        table_name VARCHAR NOT NULL,
```

Comment:
Same question as above, including the API we are interacting with, might make this status table general purpose for any of the future API import requirements as well?